### ISSUE TITLE
Windows 10 Home Ver 1909 (OS) Build 18363.815) Not working

### ISSUE THREAD

**Issue by StarfighterJ** [stascorp/rdpwrap#1063]
Windows 10 Home Ver 1909 (OS) Build 18363.815) Not working Please help
Thank You

**Comment by danmaxis** [comment]
Same thing here.... but I'm using Win10 Pro
just ran install.bat and with RDPConf.exe with administrative rights and they seem to be ok. But my computer still limited with just one user logged at a time.

**Comment by sk-lau** [comment]
Please tell us the termsrv.dll version and not the windows version

Right click on termsrv.dll --> properties --> Details --> version

**Comment by danmaxis** [comment]
My termsrv.dll is  10.0.18362.657

**Comment by sk-lau** [comment]
> My termsrv.dll is 10.0.18362.657

Did RDPConf show the green colour check  meaning it's fully supported ?

**Comment by StarfighterJ** [comment]
My termsrv.dll is 10.0.18362.657 also and RDPConf does show Green and RDPCheck works. But Microsoft Remote desk top for Android for some strange reason no longer connecting. use to work good. Now getting couldn't connect with error code 0x104.

**Comment by affinityv** [comment]
Is your Android device local to the RDP machine?  If it isn't and you are coming in via another network connection, then you may be getting blocked on the standard port (3389).  And if the latter is the case, you really should be accessing the RDP machine via a secure channel (ie ssh tunnel) and not directly.  It is very dangerous to allow RDP to be world accessible.

**Comment by danmaxis** [comment]
> > My termsrv.dll is 10.0.18362.657
> 
> Did RDPConf show the green colour check meaning it's fully supported ?

Nope, it really displayed as not supported. It was hoping that someone already solved it but it wasnt committed to the official branch.

I have no issues connecting, it's just one active user at a time that i'm trying to solve.

**Comment by StarfighterJ** [comment]
Yes my Android Device is Local to my RDP Computer. But Guess what I rememebered year or so back when everything was green and RDcheck  worked I messed with changing GUI Authentication Only to Network Level Authentication and Switched it back, and now it is working **Problem is Solved** This was a strange way to fix it Thank you all

**Comment by aroyero** [comment]
Hi, Any news? 
My termsrv.dll is 10.0.18362.657 and OS Build 18363.815 is not working. 
![image](https://user-images.githubusercontent.com/64378435/80318148-966d1c80-87cd-11ea-9470-943affb9a900.png)

please help

**Comment by aroyero** [comment]
> Hi, Any news?
> My termsrv.dll is 10.0.18362.657 and OS Build 18363.815 is not working.
> ![image](https://user-images.githubusercontent.com/64378435/80318148-966d1c80-87cd-11ea-9470-943affb9a900.png)
> 
> please help

I just copy the zip contents to c:\program files\rdp wrapper, restarted and all works fine 

![image](https://user-images.githubusercontent.com/64378435/80319857-798a1680-87d8-11ea-9a09-0906a361f477.png)

**Comment by danmaxis** [comment]
> > Hi, Any news?
> > My termsrv.dll is 10.0.18362.657 and OS Build 18363.815 is not working.
> > ![image](https://user-images.githubusercontent.com/64378435/80318148-966d1c80-87cd-11ea-9470-943affb9a900.png)
> > please help
> 
> I just copy the zip contents to c:\program files\rdp wrapper, restarted and all works fine
> 
> ![image](https://user-images.githubusercontent.com/64378435/80319857-798a1680-87d8-11ea-9a09-0906a361f477.png)

Did it runs just fine?
And by zip content you mean these files that come with rdp wrapper setup?

**Comment by aroyero** [comment]
> > > Hi, Any news?
> > > My termsrv.dll is 10.0.18362.657 and OS Build 18363.815 is not working.
> > > ![image](https://user-images.githubusercontent.com/64378435/80318148-966d1c80-87cd-11ea-9470-943affb9a900.png)
> > > please help
> > 
> > 
> > I just copy the zip contents to c:\program files\rdp wrapper, restarted and all works fine
> > ![image](https://user-images.githubusercontent.com/64378435/80319857-798a1680-87d8-11ea-9a09-0906a361f477.png)
> 
> Did it runs just fine?
> And by zip content you mean these files that come with rdp wrapper setup?

Hi,
now is working fine, yes, all files that come with the rdp wrapper

**Comment by moreleb** [comment]
Can you upload your .ini file? I have the same versions as you and it doesn't work for me.

System version: Win 10 Pro 1909 18363.815
Termsrv.dll file version: 10.0.18362.657

![prt](https://user-images.githubusercontent.com/44367715/80432687-35406880-88f5-11ea-90d1-3e5cd05333fe.jpg)

**Comment by danmaxis** [comment]
> Can you upload your .ini file? I have the same versions as you and it doesn't work for me.
> 
> System version: Win 10 Pro 1909 18363.815
> Termsrv.dll file version: 10.0.18362.657
> 
> ![prt](https://user-images.githubusercontent.com/44367715/80432687-35406880-88f5-11ea-90d1-3e5cd05333fe.jpg)

Dude, just got a solution at an other issue, it refers to this one.

This one -> [#1040](https://github.com/stascorp/rdpwrap/issues/1040 )

I proceeded as it says and worked for me.

**Comment by moreleb** [comment]
> > Can you upload your .ini file? I have the same versions as you and it doesn't work for me.
> > System version: Win 10 Pro 1909 18363.815
> > Termsrv.dll file version: 10.0.18362.657
> > ![prt](https://user-images.githubusercontent.com/44367715/80432687-35406880-88f5-11ea-90d1-3e5cd05333fe.jpg)
> 
> Dude, just got a solution at an other issue, it refers to this one.
> 
> This one -> [#1040](https://github.com/stascorp/rdpwrap/issues/1040)
> 
> I proceeded as it says and worked for me.

Thanks dude. It works

**Comment by aroyero** [comment]
Hi,
this is the .ini file. hope it helps

[rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4545750/rdpwrap.zip)

**Comment by jnstllng** [comment]
@aroyero your rdwrap.ini file was the only fiy that worked for me. Thanks...

**Comment by sebaxakerhtc** [comment]
Duplicate of https://github.com/stascorp/rdpwrap/issues/983
